x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/XPackFeatureSet.java

```diff
@@ -35,6 +36,10 @@ public Usage(StreamInput input) throws IOException {
         }
 
         public Usage(String name, boolean available, boolean enabled) {
+            Objects.requireNonNull(name);
+            if (name.isEmpty()) {
```

Comment:
Technically isBlank() would be better incase it was just whitespace, but It's fine either way as this is very unlikely since this is not user input